content/applications/finance/accounting/payables/supplier_bills/invoice_digitization.rst

```diff
@@ -16,14 +16,17 @@ created and populated based on the scanned invoices.
      <https://www.odoo.com/slides/slide/digitize-bills-with-ocr-1712>`_.
 
 .. note::
-   - The more documents you scan, the better the system identifies the correct data.
+   It is essential to review and correct (if needed) the information uploaded during digitization.
+   Then, you have to post the document by clicking on :guilabel:`Confirm`. In this manner, the AI
+   learns, and the system identifies the correct data in the future.
```

Comment:
Suggestion: I think it's rather important; could this be a separate section and not a note? Also not sure if it fits there already, it could be moved between Invoice digitization and Pricing.

Reply:
I created a new section, thanks for the suggestion! Not sure about the title...